client/src/components/Notification/NotificationItem.tsx

```diff
@@ -0,0 +1,48 @@
+import { Avatar, Grid, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+import { Notification } from '../../interface/Notification';
+import moment from 'moment';
+
+import NotificationsIcon from '@material-ui/icons/Notifications';
+
+interface Props {
+  notifications: Notification[];
+}
+
+export default function NotificationItem({ notifications }: Props): JSX.Element {
+  const classes = useStyles();
+  return (
+    <ul className={classes.listItemContainer}>
+      {notifications.map((notification) => (
+        <li className={classes.listItem} key={notification._id}>
```

Comment:
Call back (described blow) can be called here at onClick

Reply:
Same as above